### PR TITLE
feat(tags): accept tags on focusout

### PR DESCRIPTION
**Pull request by DeeMysterio**
**Before:**

Nov-15-2019 14-06-49

**Now:**

tags-focusout

### CODE REVIEW

**Review by Alchez — DISMISSED** [review]
Code looks good, just fix codacy errors and put it upstream

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/frappe/frappe/pullRequest?prid=4506519&bid=15188788)

**Comment by DeeMysterio** [comment]
> Code looks good, just fix codacy errors and put it upstream

Made to [v11](https://github.com/frappe/frappe/pull/8825) and [v12](https://github.com/frappe/frappe/pull/8826)

File: frappe/public/js/frappe/ui/tags.js [hunk 1]
```diff
@@ -34,13 +34,17 @@ frappe.ui.Tags = class {
...
+
 		this.$input.keypress((e) => {
-			if(e.which == 13 || e.keyCode == 13) {
-				const tagValue = frappe.utils.xss_sanitise(this.$input.val());
-				this.addTag(tagValue);
-				this.$input.val('');
-			}
+			if (e.which == 13 || e.keyCode == 13) callback();
 		});
+		this.$input.focusout(() => { callback(); });
```
Comment (DeveloperMetal) [inline frappe/public/js/frappe/ui/tags.js]: You can simplify this and lower memory use:

```js
this.$input.focusout(callback);
```

File: frappe/public/js/frappe/ui/tags.js [hunk 2]
```diff
@@ -34,13 +34,17 @@ frappe.ui.Tags = class {
 	}
 
 	bind() {
+		let me = this;
+		let callback = function () {
```
Comment (DeveloperMetal) [inline frappe/public/js/frappe/ui/tags.js]: You should follow modern conventions for var that should not change by accident using const:

```js
const me = this;
const callback = ...etc...
```

**Review by DeveloperMetal — COMMENTED** [review]
Code looks good, just very tiny nitpicks

**Comment by vjFaLk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/frappe/frappe/pullRequest?prid=4506519&bid=15188788)

**Review by Alchez — APPROVED** [review]

**Pull request merged**